internal/services/network/network_manager_deployment_resource.go

```diff
@@ -0,0 +1,348 @@
+package network
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonschema"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/location"
+	"github.com/hashicorp/terraform-provider-azurerm/helpers/azure"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/network/parse"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/network/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/validation"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+	"github.com/tombuildsstuff/kermit/sdk/network/2022-07-01/network"
```

Comment:
why isn't this using `go-azure-sdk`?

Reply:
find [config: Add Network 2022-09-01](https://github.com/hashicorp/pandora/pull/2316), but network RP still not show in [go-azure-sdk](https://github.com/hashicorp/go-azure-sdk/tree/main/resource-manager), do I miss something?

Reply:
@teowa The SDK likely just needed regenerating which doesn't always happen right away. It looks like [this is done now](https://github.com/hashicorp/go-azure-sdk/tree/cf00083d4c/resource-manager/network/2022-09-01) - once this has been vendored into the provider could you please update this resource to use it? Thanks!